views/eApostilles/openEApp.ejs

```diff
@@ -148,5 +148,11 @@
 
         <p>Your payment will appear on your bank statement as FCDO Legalisation Service.</p>
 
-        <a href="/download-receipt/<%= applicationId %>" class="govuk-link govuk-!-margin-top-2" role="button" download="<%= applicationId %>.pdf">Download your receipt</a>
```

Comment:
Is there validation in the download-receipt controller that processes the download requests to check the user has access to the `applcationId` they're trying to access if they were to hack the URL?

Reply:
Fair question. Yep there's validation to check the user_id in the session matches the user_id of the application in the database. Here is where the code for it is https://github.com/UKForeignOffice/loi-application-service/blob/demo/api/controllers/OpenEAppController.js#L235

Reply:
Thanks for confirming.  LGTM